app/views/teams/show.html.slim

```diff
@@ -91,7 +91,7 @@ nav.page
   ul.conferences
     - @team.members.each do |user|
       - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season, verbose: true).join(', ')}".html_safe
+        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(', ')}".html_safe
```

Comment:
Same as for the other conferences link: I would suggest to move things into the helper if it's already there.